### DISCUSSION TITLE
A problem about MOOSE mesh reading

### DISCUSSION THREAD

**Discussion by js-jixu** [idaholab/moose#20924] [Q&A General] [answered]
Hi, everyone.

I've use gmsh to generate a mesh file for input file, but MOOSE has problem in reading sidesets. The error report screenshot is here.

![error_report](https://user-images.githubusercontent.com/99813361/166093929-c4434c84-67e2-4249-9ba6-b81397ad2625.png)

When I use "--mesh-only" command to check the mesh file used by input file, it seems that the sidesets are provided. The check screenshots are provided here.
![mesh_only_1](https://user-images.githubusercontent.com/99813361/166094106-be5452e2-9f6e-4445-9a2e-da99b7a90f47.png)
![mesh_only_2](https://user-images.githubusercontent.com/99813361/166094108-633a7028-cd9b-44cd-8c6c-c837b04cae47.png)

I don't know why MOOSE has identified the sidesets but failed to read them. The MOOSE input file named framework.i, gmsh geometry file named model_geo and mesh file named model are attached, they are all txt files and can be used by modified extension name. The group constants file is also attached.

[framework.txt](https://github.com/arfc/moltres/files/8596276/framework.txt)
[model_geo.txt](https://github.com/arfc/moltres/files/8596277/model_geo.txt)
[model.txt](https://github.com/arfc/moltres/files/8596278/model.txt)
[group_constants.zip](https://github.com/arfc/moltres/files/8596279/group_constants.zip)

**Comment by js-jixu** [comment, marked as answer]
The problem is the commas in BCs block in line 152 and 163. I have to remove the commas.

**Reply by GiudGiud** [reply]
Glad you found the problem
